src/pages/index.tsx

```diff
@@ -401,19 +404,22 @@ const StyledCalloutBanner = styled(CalloutBanner)`
   }
 `
 
-const HomePage = ({ data, pageContext: { language } }) => {
+const HomePage = ({
+  data,
+  pageContext: { language = "en" },
+}: PageProps<IndexPageQuery, Context>) => {
   const intl = useIntl()
   const [isModalOpen, setModalOpen] = useState(false)
   const [activeCode, setActiveCode] = useState(0)
   const dir = isLangRightToLeft(language) ? "rtl" : "ltr"
 
-  const toggleCodeExample = (id) => {
+  const toggleCodeExample = (id: number): void => {
     setActiveCode(id)
     setModalOpen(true)
   }
   const cards = [
     {
-      image: getImage(data.robotfixed),
+      image: getImage(data.robotfixed?.childImageSharp?.gatsbyImageData),
```

Comment:
Calling `getImage(data.robotfixed)` will throw an error. `getImage` is requiring more properties related to the `Node`, which I personally don't agree it should require. I've mentioned this on their [discord](https://discord.com/channels/484383807575687178/937724599603437619/976110769450139648) but no one has responded me yet.

- [ ] Create issue in Gatsby repo regarding this and see if someone help me there.

Meanwhile, we have to do it in this way.

Reply:
https://github.com/gatsbyjs/gatsby/issues/35748